nixos/modules/services/hardware/u2f.nix

```diff
@@ -0,0 +1,31 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.u2f;
+in {
+  options = {
+    services.u2f = {
```

Comment:
Perhaps "hardware.u2f" namespace is better than "services.u2f".

Reply:
Good idea thanks